hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java

```diff
@@ -344,8 +360,6 @@
     extraClassPath
       .append("{{PWD}}")
       .append(File.pathSeparator)
-      .append(settings.getHopsLeaderElectionJarPath())
```

Comment:
@robzor92 I remember we had a funky issue and we had to add that, do you remember the specifics?

Reply:
That jar file was not on the classpath when getting the hdfs classpath --glob. I don't remember if we notified the hopsfs team at the time but we added this here to get it working. But I remember gautier telling me he added it to the classpath now.

Reply:
It should be on the classpath if it is not I'll fix it in hopsfs.